extras/bindata.bzl

```diff
@@ -40,7 +40,7 @@ def _bindata_impl(ctx):
     if not ctx.attr.modtime:
         arguments.add_all(["-modtime", "0"])
     if ctx.attr.extra_args:
-        arguments.add_all(ctx.attr.extra_args)
+        arguments.add_all(_expand_make_args(ctx, ctx.attr.extra_args))
     srcs = [f.path for f in ctx.files.srcs]
     if ctx.attr.strip_external and any([f.startswith("external/") for f in srcs]):
```

Comment:
`expand_make_variables` appears to be deprecated: https://docs.bazel.build/versions/main/skylark/lib/ctx.html#expand_make_variables

How about we use `ctx.var["BINDIR"]` and create a new attr similar to `strip_external` e.g. `strip_bindir`.

The logic could look like this:
```
if ctx.attr.strip_bindir and any([f.startswith(ctx.var["BINDIR"]) for f in srcs]):
  arguments.add("-prefix", ctx.var["BINDIR"] + "/" + ctx.label.package)
```